configurations/kalobeyei_pipeline_config.json

```diff
@@ -33,11 +33,11 @@
         "wusc_leap_s01e03_kalobeyei_activation",
         "wusc_leap_s01e04_kalobeyei_activation",
         "wusc_leap_s01e05_kalobeyei_activation",
-        "wusc_leap_s01_kalobeyei_programme_evaluation"
+        "wusc_leap_s01_kalobeyei_programme_evaluation",
+        "wusc_leap_s01e03_kalobeyei_follow_up"
```

Comment:
for consistency let s01e03_kalobeyei_follow_up come before "wusc_leap_s01_kalobeyei_programme_evaluation"

Reply:
Updated, thanks.